### PR TITLE
feat: add aave v3 debt

### PR DESCRIPTION
**Pull request by KedziaPawel**

### CODE REVIEW

File: subgraphs/enzyme-core/entities/AaveV3DebtPosition.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  aaveV3DebtPosition.active = true;
+  aaveV3DebtPosition.type = type.id;
+  aaveV3DebtPosition.eModeCategoryId = 0;
+  aaveV3DebtPosition.save();
+
+  return aaveV3DebtPosition;
+}
+
+// 255 is the max value of uint8, so we use 255 to indicate null
+// Unfortunately, it seems that we can't set eModeCategoryId to null
```
Comment (iherger) [inline subgraphs/enzyme-core/entities/AaveV3DebtPosition.ts:38]: You can set `eModeCategoryId` to null if you define it as optional in the schema.

**Review by iherger — APPROVED** [review]
Looks good!

Is there no need to listen to the events emitted by the individual `AaveV3DebtPositionLib`?

**Comment by KedziaPawel** [comment]
> AaveV3DebtPositionLib

I don't think so, we don't do it for aave v2 debt either

**Comment by KedziaPawel** [comment]
> Looks good!
> 
> Is there no need to listen to the events emitted by the individual `AaveV3DebtPositionLib`?


AaveV3DebtPositionLib

I don't think so, we don't do it for aave v2 debt either

**Pull request merged**